Default (Linux).sublime-keymap

```diff
@@ -178,14 +196,23 @@
         ]
     },
     // Bold on Alt + B
+    { "keys": ["alt+b"], "command": "run_macro_file", "args": {"file": "Packages/MarkdownEditing/macros/Transform Word - Bold - Asterisk.sublime-macro"}, "context":
+        [
+            { "key": "setting.mde.bold_marker_asterisk", "operator": "equal", "operand": true },
+            { "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
+            { "key": "selector", "operator": "equal", "operand": "text.html.markdown", "match_all": true }
+        ]
+    },
     { "keys": ["alt+b"], "command": "run_macro_file", "args": {"file": "Packages/MarkdownEditing/macros/Transform Word - Bold.sublime-macro"}, "context":
         [
+            { "key": "setting.mde.bold_marker_asterisk", "operator": "equal", "operand": false },
             { "key": "selection_empty", "operator": "equal", "operand": true, "match_all": true },
             { "key": "selector", "operator": "equal", "operand": "text.html.markdown", "match_all": true }
         ]
     },
-    { "keys": ["alt+b"], "command": "insert_snippet", "args": {"contents": "${MD_BOLD_MARKER}$1${MD_BOLD_MARKER}"}, "context":
+    { "keys": ["alt+b"], "command": "insert_snippet", "args": {"contents": "**$1**"}, "context":
```

Comment:
I may be wrong, but with a loog at the very general context of the above key bindings, it appears as this one here never gets executed.